docs/index.md

```diff
@@ -110,7 +128,7 @@ which reduces the chance of leakage.
 
 The following arguments are supported:
 
-* `region` - (Required) This is the Huawei Cloud region. It must be provided, but it can also be sourced from
+* `region` - (Optional) This is the Huawei Cloud region. It must be provided, but it can also be sourced from
```

Comment:
the description 'It must be provided' should be removed as this paramater is changed to optional

Reply:
done.